src/pages/VaccineModulePage.js

```diff
@@ -0,0 +1,281 @@
+          <View style={fridgeInfoSectionStyle}>
+            {this.renderFridgeName(fridge, isFridgeSelected)}
+            {currentTemperature !== null ? currentTemperature : null}
+
+            {isCriticalTemperature ? this.renderIcon('warning', BREACH_ICON_STYLE) : null}
+            <View style={[fridgeInfoSectionStyle, { justifyContent: 'flex-end', flexGrow: 1 }]}>
+              {this.renderFridgeExtraInfo(fridge)}
+              {this.renderFridgeStock(fridge)}
+            </View>
+          </View>
+          {isFridgeSelected && (
+            <View style={{ height: 250, alignSelf: 'stretch' }}>
+              <VaccineChart {...fridge.getTemperaturePoints()} />
+            </View>
+          )}
+        </View>
+      </View>
+    );
+  };
+
+  render() {
+    const { fridges } = this.state;
+    const { navigateTo } = this.props;
+
+    const menuButtons = [
+      { text: 'Customer Invoice', onPress: () => console.log('Customer Invoice') },
+      { text: 'Supplier Invoice', onPress: () => console.log('Supplier Invoice') },
+      { text: 'Order Stock', onPress: () => console.log('Order Stock') },
+      {
+        text: 'Manage Stock',
+        onPress: () => navigateTo('manageVaccineItems', 'Manage Vaccine Items'),
+      },
+    ];
+
+    const hasFridges = fridges.length > 0;
+    const {
+      pageContainerStyle,
+      sectionStyle,
+      imageStyle,
+      menuButtonStyle,
+      greyTextStyleLarge,
+      menuButtonTextStyle,
+    } = localStyles;
+
+    return (
+      <View style={pageContainerStyle}>
+        <View style={sectionStyle}>
+          <Image
+            style={imageStyle}
+            resizeMode="contain"
+            source={require('../images/menu_vaccines.png')}
+          />
+          {menuButtons.map(props => (
+            <Button style={menuButtonStyle} textStyle={menuButtonTextStyle} {...props} />
+          ))}
+        </View>
+        {hasFridges && fridges.map(this.renderFridge)}
+        {!hasFridges && <Text style={[greyTextStyleLarge]}>NO CONFIGURED FRIDGES</Text>}
+      </View>
+    );
+  }
+}
+
+export default VaccineModulePage;
+
+VaccineModulePage.propTypes = {
+  database: PropTypes.object.isRequired,
+  navigateTo: PropTypes.func.isRequired,
+};
+
+const localStyles = StyleSheet.create({
+  pageContainerStyle: {
+    ...globalStyles.pageContentContainer,
+    padding: 10,
+  },
+  menuButtonStyle: {
+    ...globalStyles.menuButton,
+    width: 170,
+    padding: 10,
+  },
+  menuButtonTextStyle: {
+    ...globalStyles.menuButtonText,
+    fontSize: 15,
+  },
+  sectionStyle: {
+    margin: 5,
+    padding: 10,
+    paddingRight: 15,
+    flexDirection: 'row',
+    borderColor: SHADOW_BORDER,
+    borderWidth: 1,
+    justifyContent: 'space-evenly',
+    alignItems: 'center',
+    backgroundColor: 'white',
+  },
+  extraInfoSectionStyle: {
+    flexDirection: 'row',
+    justifySelf: 'flex-end',
```

Comment:
I don't think `justifySelf` is a property

Reply:
https://github.com/openmsupply/mobile/pull/875/commits/208ccdf3dd560c3b3bdba3d27901ed53557df647